src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
The full error might be interesting, but no hurry. We could probably disable this test on FreeBSD if necessary.